src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/CcdDocument.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.model.ccd;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class CcdDocument {
```

Comment:
this should probably live here:
https://github.com/hmcts/core-case-data-store-client/tree/master/src/main/java/uk/gov/hmcts/reform/ccd/client/model